### PR TITLE
Create BIP-XXX.json - Timeless 8020

### PR DESCRIPTION
**Pull request by solarcurvey**

### CODE REVIEW

File: BIPs/BIP-212, 213.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "address", "name": "recipient", "type": "address" },
+          { "internalType": "uint256", "name": "amount", "type": "uint256" }
+        ],
+        "name": "transfer",
+        "payable": false
+      },
+      "contractInputsValues": {
```
Comment (Tritium-VLK) [inline BIPs/BIP-212, 213.json:45]: DAO Multisig should then send the BAL grant to the Timeless DAO Multisig at 0x9a8fee232dcf73060af348a1b62cdb0a19852d13

File: BIPs/BIP-212, 213.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          {
+            "internalType": "address",
+            "name": "contractAddress",
+            "type": "address"
+          }
+        ],
+        "name": "allowlistAddress",
+        "payable": false
+      },
+      "contractInputsValues": {
```
Comment (Tritium-VLK) [inline BIPs/BIP-212, 213.json:29]: Timeless will manage veBAL in the Timeless DAO multisig at 0x9a8fee232dcf73060af348a1b62cdb0a19852d13 and proposes to whitelist the Timeless Governance Multisig to interact with veBAL.

File: BIPs/BIP-212, 213.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        "contractAddress": "0xBA11E7024cbEB1dd2B401C70A83E0d964144686C"
```
Comment (Tritium-VLK) [inline BIPs/BIP-212, 213.json:66]: This looks like it is from BIP-213

Comment (Tritium-VLK) [reply BIPs/BIP-212, 213.json:66]: Yearn’s yBAL locker/voter has now been deployed…

0xBA11E7024cbEB1dd2B401C70A83E0d964144686C

https://forum.balancer.fi/t/bip-213-whitelist-yearns-ybal-voter-contract-for-vebal/4468

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]
Rename this PR/file to cover BIP-212 and 213.  I can take care of it later at some point if you're busy.

**Comment by Xeonus** [comment]
Editing BIP-ID to XXX because of delays in when this proposal is going to snapshot.

**Comment by solarcurvey** [comment]
stale

**Pull request closed without merging**